pkg/flagbuilder/build_flags.go

```diff
@@ -131,6 +131,12 @@ func BuildFlags(options interface{}) (string, error) {
 				flag = fmt.Sprintf("--%s=%s", flagName, vString)
 			}
 
+		case *string:
```

Comment:
I'm currently running this code on 1.6.1, and I've tested three cases:
- The field in the cluster spec is exists, but is empty, resulting in `--resolv-conf=`
- The field in the cluster spec exists, and is not empty, resulting in `--resolv-conf=mystring`
- The field in the cluster spec does not exist, resulting in the flag not being built.

I've also added unit tests for these three conditions in `nodeup/pkg/model/kubelet_test.go`, which indirectly test this code. If you think we need more, I can certainly add tests more specific to the `BuildFlags` function. LMK.

Also, while we're talking about this file, I want to get a second pair of eyes on this change here, since I'm still not very familiar with the ins-and-outs of Go, and reflection in Go: https://github.com/kubernetes/kops/pull/2662/files#diff-8f0a190c9dca8ca0e2ed458eb347580fR73
From my tests, this is working fine, and as expected, but I want to make sure that there's not some weird edge case I'm missing.